library/aos_version.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
```

Comment:
> Alternatively, since you are now requiring openshift_facts, there is potential for the module to exist there instead.

That's along the lines of what I did. I have a `openshift_preflight/base` (renamed from `facts`) that depends on `openshift_facts` AND includes the modules AND sets the initial value for `oo_preflight_check_results`.

`openshift_facts` takes at least 10s on an inventory with a single docker-based node. This makes the run time go from 7s to about 40s. (A single inclusion/run of `openshift_facts` in https://github.com/openshift/openshift-ansible/pull/2993/commits/c4aec0fbd112fd1c8503e935ac954da268a2fdff increased the time to 17s).

Those 10s times a multiplier is more than what I would like :(

Is there a way to avoid running the same thing twice? I definitely would not want to compose slowdowns in this way. Adding to my notes that role dependencies have some potentially dark side-effects...